scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -22,19 +22,23 @@ This notebook will demonstrate how to:
 
 In this notebook, we'll perform integration on scRNA-seq datasets from the [Single-cell Pediatric Cancer Atlas (`ScPCA`)](https://scpca.alexslemonade.org/), a database of uniformly-processed pediatric scRNA-seq data built and maintained by the Data Lab.
 This database currently hosts projects from ten different ALSF-funded labs who have generated single-cell pediatric cancer transcriptomic data with the ultimate goal of making this data easily accessible to investigators (like you!).
+**MORE TEXT HERE ON SCPCA PROCESSING**
```

Comment:
Just to be clear, this can be a sentence that says the data was mapped with alevin fry, then filtered and normalized... If you want to link to the docs, you can do that, but I don't think we need much detail at all!